Simplified/OpenEBooks/OpenEBooks_OPDS2_Catalog_Feed.json

```diff
@@ -0,0 +1,62 @@
+{
+  "catalogs": [
+    {
+      "links": [
+        {
+          "href": "https://circulation.openebooks.us/USOEI/",
+          "type": "application/atom+xml;profile=opds-catalog;kind=acquisition",
+          "rel": "http://opds-spec.org/catalog"
+        },
+        {
+          "href": "http://openebooks.net/app_user_agreement.html",
+          "type": "text/html",
+          "rel": "terms-of-service"
+        },
+        {
+          "href": "https://circulation.openebooks.us/USOEI/authentication_document",
+          "type": "application/vnd.opds.authentication.v1.0+json"
+        },
+        {
+          "href": "http://openebooks.net/app_privacy.html",
+          "type": "text/html",
+          "rel": "privacy-policy"
+        },
+        {
+          "href": "http://openebooks.net/app_acknowledgments.html",
+          "type": "text/html",
+          "rel": "copyright"
+        },
+        {
+          "href": "https://circulation.openebooks.us/USOEI/",
+          "type": "application/atom+xml;profile=opds-catalog;kind=acquisition",
+          "rel": "start"
+        },
+        {
+          "href": "https://circulation.openebooks.us/USOEI/loans/",
+          "type": "application/atom+xml;profile=opds-catalog;kind=acquisition",
+          "rel": "http://opds-spec.org/shelf"
+        },
+        {
+          "href": "https://circulation.openebooks.us/USOEI/patrons/me/",
+          "type": "vnd.librarysimplified/user-profile+json",
+          "rel": "http://librarysimplified.org/terms/rel/user-profile"
+        },
+        {
+          "href": "https://openebooks.net/",
```

Comment:
The HTTPS version of this URL isn't working -- there's a cert problem. I filed https://jira.nypl.org/browse/SEB-1604 to fix the cert and it's probably best to leave this alone rather than switching to the HTTP URL until it's fidxed.